0-getting-started/install/windows.md

```diff
@@ -14,9 +14,11 @@ _Prerequisites:_ We provide native 64-bit binaries for Windows 7 and above. A 64
 [Download](https://download.rethinkdb.com/windows/rethinkdb-{{site.version.full}}.zip) the ZIP archive and unpack it in a directory of your choice.
 
 {% infobox alert %}
-The Windows port of RethinkDB is still in beta, and is not considered production ready yet!
+The Windows port of RethinkDB is a recent addition and hasn't received as much tuning as the Linux and OS X versions yet. Please report any performance issues on [GitHub][gh-issues].
 {% endinfobox %}
 
+[gh-issues]: https://github.com/rethinkdb/rethinkdb/issues/
```

Comment:
This will probably need to be moved into the infobox; that's a custom tag, and everything in the `{% infobox %} ... {% endinfobox %}` block has to be self-contained.

You can also consider removing the `alert` so it's just a normal infobox now.